### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by FallBackITA27** [Byron/google-apis-rs#408] [Q&A]
Please, update the google_sheets4 documentation.
I've been working with it and it's pain.
All of a sudden there's a required parameter to write, called "request", with no description provided, I accidentally found it on the main page and the most I could gather is that it's of type "ValueRange", which there's no constructor for.

Partly it's my fault for not coding mainly in Rust but it's really an headache to work with :P

**Comment by Byron** [comment]
Could you be more specific as to which configuration you are referring to?

**Reply by FallBackITA27** [reply]
Could you explain what you mean by configuration?

**Reply by Byron** [reply]
Oh, I mean documentation. A URL would do the trick.
